publish/models/version.py

```diff
@@ -80,6 +83,10 @@ def make_version(cls, dandiset: Dandiset = None) -> str:
     def from_girder(cls, dandiset: Dandiset, client: GirderClient) -> Version:
         draft_folder = client.get_json(f'folder/{dandiset.draft_folder_id}')
 
-        version = Version(dandiset=dandiset, metadata=draft_folder['meta'])
+        metadata = draft_folder['meta']
+        name = metadata['dandiset'].pop('name')
```

Comment:
I'd like to have a test case to see what happens if you include a non-string value in one of these metadata fields.

Reply:
I added a test case where `name` and `description` are of type `int` and `dict`, respectively, in https://github.com/dandi/dandi-publish/pull/63/commits/bea9826e2832157a09cd9cd6c15ba3bcbf6ca3d6. I had to add some extra stuff to the mocked girder client since there was no way to directly modify the `metadata` field of the mocked girder draft folder. 
It seems that the Django ORM automatically casts other data types to strings for `CharField` and `TextField` fields, so a non-string value doesn't cause any errors (unless the string representation of whatever type it is happens to go over the `max_length` in the case of `CharField`). I also verified this without using `Faker` by manually setting `name` and `description` to non-string values directly in the version model's `from_girder` method. Knowing this I'm not sure if a test would still be useful, what do you think?

Reply:
Thanks for the investigation. As long as Django handles the coercion without errors, I'm fine to allow other types to be implicitly accepted. No test is necessary, imo.

Also, it's possible (I'm not certain) that once we use a DRF Serializer to do the coercion, it may be more strict, but that's a separate matter.

Reply:
Agreed. I removed the test in https://github.com/dandi/dandi-publish/pull/63/commits/7117a04d5ec021fc59e39c25f3d6539e39fce5b6